x/metadata/client/cli/tx.go

```diff
@@ -406,7 +601,7 @@ $ %s tx metadata recspec1qh... recordname inputname1,typename1,hashvalue;inputen
 				return err
 			}
 
-			resultType := definitionType(args[4])
+			resultType := types.DefinitionType(types.DefinitionType_value[fmt.Sprintf("DEFINITION_TYPE_%s", strings.ToUpper(args[4]))])
```

Comment:
lots of chance for stuff to go wrong in this like ... that map lookup could be checked for an invalid result... on the other hand if this simply blows up and returns to the console that may be sufficient

Reply:
Validate basic should error on unspecified as it is not a valid option to use ...